3rdparty/jpeg-compressor/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_library(jpgd STATIC ${CMAKE_CURRENT_SOURCE_DIR}/jpgd.h ${CMAKE_CURRENT_SOURCE_DIR}/jpgd.cpp)
+target_include_directories(jpgd
+    PUBLIC
+    ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
This path should be quoted, I think. They might contain spaces and/or semicolons.